doc/01_installation.md

```diff
@@ -277,11 +281,18 @@ You must have:
 - A schema: `test`
 - A table: `test.simple_sensor`
 
+Hint: you can get a PG command interface in the container using `docker exec -it <container name> psql -U postgres` in the host
+
 ```sql
+-- Assuming database 'tests' already exist, typically CREATE DATABASE tests;
+\c tests
 CREATE SCHEMA IF NOT EXISTS test;
 CREATE TABLE test.simple_sensor (
   recvtime TIMESTAMPTZ,
+  fiwareservicepath TEXT,
   entityid TEXT,
+  entitytype TEXT,
+  timeinstant TIMESTAMPTZ,
   temperature DOUBLE PRECISION
```

Comment:
Both (i.e. database creation in advanced and right columns for the table) are needed, as far as I checked.